.github/labeler.yml

```
@@ -0,0 +1,18 @@
+documentation:
+  - changed-files:
+      - any-glob-to-any-file: ['docs/**', 'mkdocs.yml']
+docker:
+  - changed-files:
+      - any-glob-to-any-file: ['**/Dockerfile', 'docker-compose.yml']
+element:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/elements/**']
+python:
+  - changed-files:
+      - any-glob-to-any-file: ['apps/prairielearn/python/**']
+github_actions:
+  - changed-files:
+      - any-glob-to-any-file: ['.github/**']
```

The `.github` directory contains other things unrelated to Actions, e.g. Dependabot config and your new issue templates.

```suggestion
      - any-glob-to-any-file: ['.github/workflows/**']
```

---

Hmm -- what if I use the `tooling` label instead? I would want something that labels those changes instead.